Description: add Show instance to build with ghc 7.4
 Add Show instance for ToSElem.
 A newer upstream version has these same changes already.
Author: Joey Hess <joeyh@debian.org>

--- haskell-hstringtemplate-0.6.6.orig/Text/StringTemplate/Instances.hs
+++ haskell-hstringtemplate-0.6.6/Text/StringTemplate/Instances.hs
@@ -83,7 +83,7 @@ instance ToSElem Int where
 instance ToSElem Integer where
     toSElem = STR . show

-instance Integral a => ToSElem (Ratio a) where
+instance (Integral a, Show a) => ToSElem (Ratio a) where
     toSElem = STR . show

 --Dates and Times
--- haskell-hstringtemplate-0.6.6.orig/Text/StringTemplate/Base.hs
+++ haskell-hstringtemplate-0.6.6/Text/StringTemplate/Base.hs
@@ -536,7 +536,7 @@ functn = do



-mkIndex :: Num b => [b] -> [[SElem a]]
+mkIndex :: (Num b, Show b) => [b] -> [[SElem a]]
 mkIndex = map ((:) . STR . show . (1+) <*> (:[]) . STR . show)
 ix0 :: [SElem a]
 ix0 = [STR "1",STR "0"]
